packages/kamino-sdk/src/Kamino.ts

```diff
@@ -1688,6 +1692,38 @@ export class Kamino {
     };
   };
 
+  /**
+   * Get an instruction to open liquidity position for a Kamino strategy based on onchain params
+   * @param strategy strategy you want to open liquidity position for
+   * @param positionMint position mint pubkey
+   * @param status strategy status
+   */
+  openPositionFromPercentageRebalanceParameters = async (
```

Comment:
this needs to be generic, not just for a position type and also needs to take the fields as input rather than the strategy such that it can be called at the same time, not only after the strategy has finished being created (cause you use async fetching).

`getOpenPositionIx()` basically and is generic and it uses the fields as much as possible, get the generic poolInfo as args so you can do it all sync, not async